src/components/Editor/MonacoWrapper.tsx

```diff
@@ -52,7 +52,7 @@ export const MonacoWrapper: React.FunctionComponent<MonacoWrapperProps> = ({
   const onChange = debounce((v: string) => {
     EditorService.updateState({ content: v });
     SpecificationService.parseSpec(v);
-  }, 250);
```

Comment:
@magicmatatjahu any reason this was changed to 625? Was it to help with performance?

Reply:
I noticed that 250 msc is not that long time for garbarge collector to remove old references to old VDOM from React. those 625 are not that long time again but some references are able to remove themselves and free up memory faster :)

Reply:
Ah o wow, cool thanks for explaining :)